### PR TITLE
feat(query): support of Terraform modules for "S3 Bucket Without Enabled MFA Delete"

### PR DESCRIPTION
**Pull request by rafaela-soares**
**Proposed Changes**
- Support of Terraform modules for "S3 Bucket Without Enabled MFA Delete"

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/aws/s3_bucket_without_enabled_mfa_delete/test/negative2.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+resource "aws_s3_bucket" "positive6" {
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/s3_bucket_without_enabled_mfa_delete/test/negative2.tf:1]: @rafaela-soares why are you repeating the exact same samples we already have for the resource `aws_s3_bucket` and not including any samples related to the module query update?

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
You have duplicate samples for some reason

File: assets/queries/terraform/aws/s3_bucket_without_enabled_mfa_delete/test/negative2.tf (hunk 1 above)
Comment (rafaela-soares) [reply assets/queries/terraform/aws/s3_bucket_without_enabled_mfa_delete/test/negative2.tf:1]: My bad, I am sorry. I thought I was using a module template.

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**